src/formatDate.js

```diff
@@ -8,7 +8,35 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const delimiters = ['-', '/', '.'];
```

Comment:
Hardcoding delimiters is not flexible if new delimiters are introduced in the future. Consider deriving the input delimiter directly from the `fromFormat` array.